frontend/src/components/Reports/routine/ReferredOut.js

```diff
@@ -80,7 +86,7 @@ const ReferredOut = () => {
     setLoading(true);
     const baseParams = 'report=referredOut&type=patient';
     const baseUrl = `${config.serverBaseUrl}/ReportPrint`;
-    const url = `${baseUrl}?${baseParams}&upperDateRange=${reportFormValues.startDate}&lowerDateRange=${reportFormValues.endDate}&locationCode=${locationCodes[0].id}`;
+    const url = `${baseUrl}?${baseParams}&upperDateRange=${reportFormValues.startDate}&lowerDateRange=${reportFormValues.endDate}&locationCode=${selectedLocationCode}`;
```

Comment:
an example of this is where we add the id to the url rather than the value -> https://localhost/api/OpenELIS-Global/ReportPrint?report=referredOut&type=patient&upperDateRange=05%2F03%2F2024&lowerDateRange=08%2F03%2F2024&locationCode=2500

Reply:
@mherman22 , 
`upperDateRange=${reportFormValues.endDate}`  not   `upperDateRange=${reportFormValues.startDate}`
and so is 
l`owerDateRange=${reportFormValues.sartDate}` not `lowerDateRange=${reportFormValues.endDate}`

Reply:
Oh makes perfect sense